### DISCUSSION TITLE
Firmware update on SFP Technicolor module

### DISCUSSION THREAD

**Discussion by gianry** [Anime4000/RTL960x#298] [Q&A]
I 've a RTL 9601CI  gpon module and I need to use it with FTTH 2.5 gigabit. Is it possible to update firmware  on web gui ith the file V2801F_V1.9.0-240204 ?

thanks

**Comment by gianry** [comment]
![IMG_3320](https://github.com/Anime4000/RTL960x/assets/121199187/d2c5be43-a9d2-467d-9a25-c9ff52d92f71)
![IMG_3319](https://github.com/Anime4000/RTL960x/assets/121199187/9b795b98-38fd-4e29-8688-f4713362d3b5)

**Reply by Anime4000** [reply]
I not sure, try flash original VSOL firmware first,
make sure set `PON_MODE 3` to prevent invalid key bootloop

**Reply by gianry** [reply]
is this firmware good to set LAN SDS MODE to 4 or 6 for enabling 2.5 gigabit ethernet link ?

**Reply by Anime4000** [reply]
not yet, at least not broken when flash it, I can't be certain if works like V2801F and TWCGPON657

**Reply by gianry** [reply]
> not yet, at least not broken when flash it, I can't be certain if works like V2801F and TWCGPON657

could you post me the link for this original Vsol firmware you wrote ?
 I can try the modified firmware in any way as the GPON module is for testing and it is not usually used.
